include/libdnf/rpm/sack.hpp

```diff
@@ -61,11 +68,53 @@ inline ReldepId::ReldepId(int id) : id(id) {}
 class Package;
 class Reldep;
 class ReldepList;
+class Repo;
 
 class Sack {
```

Comment:
What about to name it `SolvSack`?

Reply:
Yes, we agreed on the renaming to `SolvSack` some time ago. Question was who will rename it. So I'll do it in this PR.